src/Aframe/arcball-controller.js

```diff
@@ -17,8 +17,8 @@ THREE.TrackballControls = function (object, domElement) {
       'THREE.TrackballControls: "document" should not be used as the target "domElement". Please use "renderer.domElement" instead.'
     );
 
-  var _this = this;
-  var STATE = {
+  let _this = this;
```

Comment:
If you use arrow functions `() => {}` the `this` is preserved, meaning you don't need to save `_this` and use it in the closures

Related, I'd consider getting rid of those self-invoking functions like `getMouseOnCircle`. I get that they're trying to hid implementation variables from the rest of the code, but that stuff is hard to read and hard to work with when things go wrong

Reply:
I'm going to pull this into issue #61 and work on it separately